recipes/hiclass/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "hiclass" %}
+{% set version = "2.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9d8f6ec914ed5dec2812ccc19417246f5e08f05ee250d04af74cac450f175ddd
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - hiclass = hiclass.__main__:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - networkx
+    - numpy
+    - scikit-learn
+
+test:
+  imports:
+    - hiclass
+  requires:
+    - pip
+  commands:
+    - pip check
+
+about:
+  home: https://gitlab.com/dacs-hpi/hiclass
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE
+  summary: 'Hierarchical Classification Library'
+  description: |
+    HiClass is a library for hierarchical classification which
+    is inspired in scikit-learn's usability and is optimized
+    for speed. It is pure python code compatible with
+    Python 3.8+ and requires networkx, numpy and scikit-learn.
+  doc_url: https://hiclass.readthedocs.io/
+  dev_url: https://gitlab.com/dacs-hpi/hiclass
+
+extra:
+  recipe-maintainers:
+    # GitHub IDs for maintainers of the recipe.
```

Comment:
You can remove this comment.

Reply:
Thank you for the suggestion! This comment slipped by, but I will remove it. Since this PR was merged, should I open a new PR to remove this comment?

Reply:
Yep. You can do it in the feedstock. That was not a blocker to merge it here.